mobify/platform-scaffold
# WEB-1280: a2hs improvements

This PR contains changes to the headerbar when the user is viewing the page in A2HS mode. 
Each piece of this PR has been reviewed separately, so we just need a sanity check review here. 

 **JIRA**: https://mobify.atlassian.net/browse/WEB-1280
 **Linked PRs**: n/a

## Changes
- Update styling for headerbar in A2HS mode [960](https://github.com/mobify/platform-scaffold/pull/960)
- Added More menu to headerbar in A2HS mode [973](https://github.com/mobify/platform-scaffold/pull/973)

## Todos
- [ ] SDK preview release, update scaffold with new version

## How to test-drive this PR
- checkout the `develop` branch of the SDK
- cd into the SDK folder
- run `npm link`
- cd into the scaffold folder
- run `npm link progressive-web-sdk`
- run `npm run dev`
- Preview [Merlin's Potions](https://preview.mobify.com/?url=https%3A%2F%2Fwww.merlinspotions.com%2F&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1)
- browse to https://www.merlinspotions.com/?homescreen=1
- the header bar should appear and function as described in the JIRA ticket
